libraries/typescript/Microsoft.Bot.Builder.Adapters.WeChat/src/accessTokenStorage.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * @module wechat
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Storage, StoreItems } from 'botbuilder-core';
+import { WeChatAccessToken } from './weChatSchema';
+
+/**
+ * Storage provider for access token.
+ */
+export class AccessTokenStorage {
+    private Storage: Storage;
```

Comment:
>Storage [](start = 12, length = 7)

variables are always camelCase in JavaScript. The only thing that's PascalCase are Type names.